packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -324,6 +332,46 @@ export function getPublishResolver<TSource, TContext, TArgs>({
   }
 }
 
+/**
+ * Returns values for link arguments, also covers the cases for
+ * if the link parameter contains constants that are appended to the link parameter
+ *
+ * e.g. instead of:
+ * $response.body#/employerId
+ *
+ * it could be:
+ * abc_{$response.body#/employerId}
+ */
+function inferLinkArguments<TSource, TContext, TArgs>({
+  paramName,
+  value,
+  resolveData,
+  source,
+  args
+}: inferLinkArgumentsParam<TSource, TContext, TArgs>)  {
+  if (typeof value === 'object') {
+    return Object.entries(value).reduce((acc, [key, value]) => {
+      acc[key] = inferLinkArguments({paramName, value, resolveData, source, args})
+      return acc
+    }, {})
+  }
+
+  if (typeof value !== 'string') {
```

Comment:
As well as a check here. However, I am still working on finishing a test but unfortunately I don't think I will be able to finish this today and I really want to get this checked in. I think for our purposes, this PR is good enough to get merged in!